src/components/admin/security/LoginAttemptsMap.tsx

```
@@ -0,0 +1,167 @@
+'use client';
+
+import React, { useState, useRef, useCallback } from 'react';
+import Map, { Marker, Popup, NavigationControl } from 'react-map-gl';
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)